google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -898,10 +898,6 @@ def default_acl
         # @see https://cloud.google.com/storage/docs/json_api/v1/buckets/setIamPolicy
         #   Buckets: setIamPolicy
         #
-        # @param [Boolean] force Force load the latest policy when `true`.
```

Comment:
Do we use `force: true` for these semantics in other Google Cloud Client Libraries today?

For a bit of background, this conversation started with a code review I was doing for Frank for Bucket IAM samples.

I saw this line of code: `@bucket.policy force: true` and [commented](https://github.com/GoogleCloudPlatform/ruby-docs-samples/pull/167#discussion_r111654037) that I couldn't, for the life of me, guess what it did.  Like *"What does it mean to 'force' a policy?  Or... what!?  I can't even..."*

I like `policy.reload!` because of its idiomatic nature (read: Rails).  If a param is also available, I would much prefer `bucket.policy reload: true` because it has semantic meaning and I can guess/infer from the language what the code does.